sdl_viewer/src/bin/sdl_viewer.rs

```diff
@@ -11,16 +11,11 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-
-use sdl_viewer::SdlViewer;
+use point_viewer::octree::OctreeFactory;
+use sdl_viewer::run;
 
 fn main() {
-    SdlViewer::new()
-        .register_octree_factory("grpc:".into(), |p| {
-            Ok(Box::new(
-                point_viewer_grpc::octree_from_address(p.as_str())
-                    .expect("Could not create octree."),
-            ))
-        })
-        .run();
+    let octree_factory = OctreeFactory::new()
+        .register_octree_factory("grpc:", |p| point_viewer_grpc::octree_from_address(p));
```

Comment:
`.register_octree_factory("grpc:", |p| point_viewer_grpc::octree_from_address(p));`
->
`.register_octree_factory("grpc:", point_viewer_grpc::octree_from_address);`

Reply:
cargo fmt complained about that..  that's the proposed formatting

Reply:
cargo fmt was complaining about that... that's the proposed syntax

Reply:
Hm, for me it didn't complain. Did you actually mean cargo clippy? Which version of rust was complaining?

Reply:
I confused the line with a similar one in avmapping geometry, I double checked and works